values/nginx-ingress/nginx-ingress.gotmpl

```diff
@@ -80,21 +87,23 @@ controller:
       additionalLabels: # needed to be picked up by our one and only prometheus-operator:
         prometheus: system
   service:
-    # externalTrafficPolicy: Local
+    externalTrafficPolicy: Local
```

Comment:
no, I don't wish to add links to external docs, because we have to stay consistent. Just googling `"ingress nginx externalTrafficPolicy" will get you there

Reply:
did you read those docs? then your question is answered